src/ErrorHandler.php

```diff
@@ -337,4 +295,32 @@ private function cleanBacktraceFromErrorHandlerFrames(array $backtrace, string $
 
         return $cleanedBacktrace;
     }
+
+    /**
+     * @param \ErrorException $errorAsException The error to be passed onto listeners
+     */
+    private function invokeErrorListeners(\ErrorException $errorAsException): void
+    {
+        foreach ($this->errorListeners as $listener) {
+            try {
+                $listener($errorAsException);
```

Comment:
Shouldn't we use `call_user_func` here?

Reply:
It's basically the same. We even skip a function invocation like this.

Reply:
Since we switched to `callable` we now have to use `call_user_func`

Reply:
I've added a test that proves that `call_user_func` is not needed.